specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimapis.json

```diff
@@ -935,6 +1792,9 @@
           {
             "$ref": "#/parameters/ApiIdParameter"
           },
+          {
+            "$ref": "#./apimtags.json#/parameters/TagIdParameter"
```

Comment:
>#./ [](start = 21, length = 3)

you can fix this and run the tool [https://github.com/Azure/oav](https://github.com/Azure/oav) locally before submitting again.

Reply:
fixed